edx_sga/utils.py

```diff
@@ -0,0 +1,16 @@
+"""
+Utility functions for the SGA XBlock
+"""
+import datetime
+import pytz
+
+from django.conf import settings
+
+
+def tznow():
+    """
+    Get current date and time.
+    """
+    return datetime.datetime.utcnow().replace(
+        tzinfo=pytz.timezone(getattr(settings, "TIME_ZONE", pytz.utc.zone))
```

Comment:
i think we should improve this according to @noisecapella suggestion
https://github.com/mitodl/edx-sga/pull/169#issuecomment-339054815

Reply:
This was the existing logic in SGA. I simply moved this function into a utils file. If we want to change this logic, I would probably say it's out of scope for this PR.

Reply:
Feel free to open a new issue